source/Integrate/Tutorials/analytics_with_keen_io.md

```diff
@@ -9,64 +9,65 @@ layout: page
-
-Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY
-```
-
-If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
-
-```
-https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY
-
- ```
-
-In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
+2. Enter the Keen HTTP POST URL, exampled below, as the endpoint where SendGrid will send event notifications. You can find your `KEEN_PROJECT_ID` and `KEEN_API_WRITE_KEY` listed in the [Access tab](https://keen.io/projects) of your Keen project.
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v3?api_key=KEEN_API_WRITE_KEY```
+    If you're using Version 1 or 2 of the SendGrid Event API, use this as your Post Event URL instead:
+    ```https://api.keen.io/3.0/projects/KEEN_PROJECT_ID/partners/sendgrid/versions/v2?api_key=KEEN_API_WRITE_KEY```
+3. In the settings, select the events you want to post to Keen IO (why not all of them?). To do this, check "Select All" under the "Select Actions", or select just the ones you'd like to send to Keen IO.
 
 That’s it! Now, as your emails make their way through SendGrid, all of the event information will be posted to Keen IO. A new Event Collection, aptly named "SendGrid Email Events", will be created within your project.
 
 
-**Step 3: Send a test email!**
+{% anchor h4 %}
+Step 3: Send a test email!
+{% endanchor h4 %}
 
-To test this out, you can click on the “Test Your Integration” button in the Event Notification settings inside of SendGrid, which should create a set of sample email events from SendGrid into your Keen IO Project.
+To test this out, you can click on the “Test Your Integration” button in the Event Notification settings inside of SendGrid, which creates a set of sample email events from SendGrid into your Keen IO Project.
 
 Your SendGrid email data should now be populated in Keen IO. To check, head over to your Keen IO project page. Check out the "Streams" tab, select "SendGrid Email Events" from the drop-down list, and then you can see your Last 10 Events streamed. This should show at least one of our delivered email events, or more if you went to your inbox and opened it!
 
-## <a name="-analyze-your-sendgrid-data"></a>Analyze Your SendGrid Data
+{% anchor h3 %}
+Analyzing Your SendGrid Data
+{% endanchor h3 %}
 
-**Step 1: Log into your Keen account, select your Project**
+{% anchor h4 %}
+Step 1: Log into your Keen account, select your Project
```

Comment:
Re to next few comments - I actually did this for the first round, and it didn't look very good. There's too much going on in each section for it not to break md lists, so I'm going to leave it for now.